core/controllers/admin.py

```diff
@@ -848,3 +848,23 @@ def post(self):
             self.render_json({})
         else:
             raise self.InvalidInputException('This app cannot send emails.')
+
+
+class InteractionsByExplorationId(base.BaseHandler):
+    """Handler to retrive the list of interaction id's for a exploration id."""
+    GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+
+    @acl_decorators.can_access_admin_page
+    def get(self):
+        exploration_id = self.request.get('exploration_id', None)
+        if exploration_id is None:
```

Comment:
Whenever it tries to run this test it skips this condition probably because I'm getting the exploration_id like u'None so which method should I use to covert this JSON.

Reply:
You can try calling this API without any param, maybe? [Also, check why it's 'None' not `None`, check other test files, and see how it's written there.]

Reply:
Tried out empty params it's gives 500 would that be okay ?

Reply:
Nope, never expect API to return 500. Check what's going on, you can try this to see the error log:

Move the handler functionality inside try except block and print the exception to see what's the error log*

Reply:
Thank You i was able to solve it and I guess the PR is now complete :)

Reply:
@DubeySandeep It took me a lot of time to complete a simple task but I guess it was worth it and I got to know about a couple of things which will help me throughout the project :)